packages/@react-spectrum/textfield/intl/en-US.json

```diff
@@ -0,0 +1,3 @@
+{
+  "valid": "Valid"
```

Comment:
Will need to consult accessibility on the best string here. The invalid announced "invalid text" on MacOS VO.